src/arbitrum/epoch-based-managed-vaults/stratagies/BaseStrategy.sol

```diff
@@ -84,6 +80,7 @@ abstract contract BaseStrategy is ReentrancyGuard, IStrategy {
     /// @inheritdoc IStrategy
     function isActive() public view virtual returns (bool) {
         if (isStrategiesActiveOverride) return false;
+        if (IERC20(assetVaultPrimaryAsset).balanceOf(address(this)) > 0) return true;
```

Comment:
What is the definition of "active" with managed vaults? Epoch based strategies may not do anything even though the only purpose for inactivity is to deposit assets. Mapping this to Y2K, the weekly vaults are active between Wednesdays and Sunday, while Monday and Tuesday nothing happens other than people depositing into vaults and waiting for the epoch to start and vaults to trade actively.

Reply:
Definition of "active" is if value is in the vault. If you look at it from the opposite direction, an epoch can be finished only when no AssetVault (and so also StrategyVault) is active, i.e. when all assets/value is back in the MetaVault.